packages/react-examples/src/react-tooltip/Tooltip/Tooltip.stories.tsx

```diff
@@ -99,57 +101,87 @@ export const Positioning = () => {
         </Tooltip>
         <Tooltip content="before center" position="before" align="center">
-          <button className={styles.target} style={{ gridArea: '3 / 1' }} />
+          <button style={{ gridArea: '3 / 1' }}>before center</button>
         </Tooltip>
         <Tooltip content="before bottom" position="before" align="bottom">
-          <button className={styles.target} style={{ gridArea: '4 / 1' }} />
+          <button style={{ gridArea: '4 / 1' }}>before bottom</button>
         </Tooltip>
 
         <Tooltip content="after top" position="after" align="top">
-          <button className={styles.target} style={{ gridArea: '2 / 5' }} />
+          <button style={{ gridArea: '2 / 5' }}>after top</button>
         </Tooltip>
         <Tooltip content="after center" position="after" align="center">
-          <button className={styles.target} style={{ gridArea: '3 / 5' }} />
+          <button style={{ gridArea: '3 / 5' }}>after center</button>
         </Tooltip>
         <Tooltip content="after bottom" position="after" align="bottom">
-          <button className={styles.target} style={{ gridArea: '4 / 5' }} />
+          <button style={{ gridArea: '4 / 5' }}>after bottom</button>
         </Tooltip>
 
         <Tooltip content="below start" position="below" align="start">
-          <button className={styles.target} style={{ gridArea: '5 / 2' }} />
+          <button style={{ gridArea: '5 / 2' }}>below start</button>
         </Tooltip>
         <Tooltip content="below center" position="below" align="center">
-          <button className={styles.target} style={{ gridArea: '5 / 3' }} />
+          <button style={{ gridArea: '5 / 3' }}>below center</button>
         </Tooltip>
         <Tooltip content="below end" position="below" align="end">
-          <button className={styles.target} style={{ gridArea: '5 / 4' }} />
+          <button style={{ gridArea: '5 / 4' }}>below end</button>
         </Tooltip>
       </div>
     </>
   );
 };
 
+export const ControlledTooltip = () => {
+  const [tooltipVisible, setTooltipVisible] = React.useState(false);
+  return (
+    <>
+      <Tooltip content="The visibility of this tooltip is controlled by the parent component" visible={tooltipVisible}>
+        <button onClick={() => setTooltipVisible(v => !v)}>Toggle tooltip</button>
+      </Tooltip>
+    </>
+  );
+};
+
 export const OnlyIfTruncated = () => {
+  const textContainerRef = React.useRef<HTMLDivElement>(null);
+  const [tooltipVisible, setTooltipVisible] = React.useState(false);
   const [wide, setWide] = React.useState(true);
-  const text = 'The tooltip will only show if the text is truncated.';
-
+  const text = 'The tooltip only shows if the text is truncated';
   return (
     <>
-      <Tooltip content={text} onlyIfTruncated>
+      <button onClick={() => setWide(w => !w)}>Toggle container width</button>
+      <Tooltip
+        content={text}
+        visible={tooltipVisible}
```

Comment:
nit: either you rename this to `open` or I rename the prop in `Menu` and `Popover` to `visible`  for consistency. Your call

Reply:
I thought about `open` for tooltip, but it doesn't really make sense in the context of tooltips. I don't necessarily think this prop name has to be aligned with the menu and popover prop though: `open` makes more sense in the context of those components.

That said, I could be convinced to rename this to `open` if you feel strongly about the consistency.

Reply:
The key factor for me here is that all of these components use popper. I think as a user of the suite it would be far easier just to know that `open` `onOpenChanged` can be used interchangeably.

Perhaps it might change as we get more components, but I guess the current `Dropdown` or `Select` would have a similar concept.

Reply:
IMO popper is an implementation detail, and is about layout more than functionality. I don't think that should be a factor in the API design.

Also, Tooltip is conceptually quite different from Menu, Popover, Dropdown, etc. I'm not sure that there'd be an expectation that the prop names would align between them. To me, `open` makes sense with an interactive component, but tooltips are purely informational.

Reply:
That's fair, let's leave this up to the API review that we discussed and see the results of that